dev_utils/env.sync

```diff
@@ -0,0 +1,10 @@
+ARCHIVE_TYPE=s3
+ARCHIVE_URL=https://s3
+BACKUP_TYPE=s3
+BACKUP_URL=https://s3
+BROKER_EXCHANGE=sda
+BROKER_HOST=mq
+BROKER_QUEUE=backup
+BROKER_ROUTINGKEY=accessionIDs
```

Comment:
```suggestion
BROKER_QUEUE=accessionIDs
BROKER_ROUTINGKEY=backedup
```
Having both sync and finalize listen to `accessionIDs` by default makes for less overall changes.
And since this is only a configuration set on startup it's actually irrelevant what the queue/routing keys are.

Reply:
@jbygdell should I merge the pull request as is then?